src/content/blog/astro-350.mdx

```diff
@@ -0,0 +1,264 @@
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  prefetch: true
+})
+```
+
+In addition to being a core feature, the new prefetch also comes with more flexibility:
+
+- You can define if prefetch occurs on `tap` (a touch that's not a full click), `hover` and `viewport` (when the element is in view).
+- You can define which strategy is used on a per-link basis using the `data-astro-prefetch` attribute.
+- You can specify that *all links* are prefetched by default (the existing behavior when using `<ViewTransitions />`).
+- The new prefetch will ignore hovers/scroll that occur quickly, to prevent overfetching content the user is not likely to click.
+
+For `<ViewTransitions />`, the new prefetch also makes it possible to opt-out of prefetch on a per-link basis, something that was not previously possible:
+
+```js
+<a href="/logout" data-astro-prefetch="false">Logout</a>
+```
+
+Visit the [Prefetch guide](https://docs.astro.build/en/guides/prefetch/) for more information.
+
+## Form support in View Transitions router
+
+The `<ViewTransitions />` router can now handle form submissions, allowing the same animated transitions and stateful UI retention on form posts that are already available on `<a>` links. With this addition, your Astro project can have animations in all of these scenarios:
+
+- Clicking links between pages.
+- Making stateful changes in forms (e.g. updating site preferences).
+- Manually triggering navigation via the `navigate()` API.
+
+This feature is opt-in for semver reasons and can be enabled by adding the `handleForms` prop to the `<ViewTransitions />` component:
+
+```astro
+---
+// src/layouts/MainLayout.astro
+import { ViewTransitions } from 'astro:transitions';
+---
+
+<html>
+  <head>
+    <!-- ... -->
+    <ViewTransitions handleForms />
+  </head>
+  <body>
+    <!-- ... -->
+  </body>
+</html>
+```
+
+Just as with links, if you don't want the routing handling a form submission, you can opt out on a per-form basis with the `data-astro-reload` property:
+
+```astro
+---
+// src/components/Contact.astro
+---
+<form class="contact-form" action="/request" method="post" data-astro-reload>
+  <!-- ...-->
+</form>
+```
+
+Form support works on post `method="get"` and `method="post"` forms.
+
+## Image optimization improvements
+
+Astro assets has continued to under go improvements and in 3.5 we now have:
+
+- Original images are deleted from the final build that are not used outside of the optimization pipeline. For users with a large number of these images (e.g. thumbnails), this should reduce storage consumption and deployment times.
+- A new property, `propertiesToHash`, has been added to allow specifying which properties of `getImage()` / `<Image />` / `<Picture />` should be used for hashing the result files when doing local transformations. For most services, this will include properties such as `src`, `width` or `quality` that directly changes the content of the generated image.
```

Comment:
comma nit
```suggestion
- A new property, `propertiesToHash`, has been added to allow specifying which properties of `getImage()` / `<Image />` / `<Picture />` should be used for hashing the result files when doing local transformations. For most services, this will include properties such as `src`, `width`, or `quality` that directly change the content of the generated image.
```